extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -439,6 +445,9 @@ export class AmpA4A extends AMP.BaseElement {
       }
     }
 
+    this.detectedFont_ =
```

Comment:
That's assuming this parameter is unique to AdSense. If it's required in DoubleClick, then there will be some code duplication. (Why isn't there a common class for adsense-impl and doubleclick-impl to inherit from?)